packages/features/calendars/DatePicker.tsx

```diff
@@ -414,6 +423,7 @@ const DatePicker = ({
           slots={!isBookingInPast ? slots : {}}
           includedDates={!isBookingInPast ? includedDates : []}
           isBookingInPast={isBookingInPast}
+          timezone={timezone}
```

Comment:
Instead of passing the timezone, can we set or modify the minDate from here?

Reply:
Explained, refer other comments.